0300-method.Rmd

```diff
@@ -28,14 +28,13 @@ Sites were surveyed using a hip chain to track distance with the following attri
 A clinometer was used to estimate gradient, channel measurements were taken using a tape measure and water depths measured using a measuring stick.  Photographs were taken of representative habitats throughout the surveyed reaches.  All field measurements were recorded on Level 1 Habitat Survey Data Form templates (@johnston_slaney1996FishHabitat). GPS waypoints were collected at the beginning and end of all Primary habitat units to allow georeferenceing of field data.
 
 ## Stream Discharge
-Hydrometric transects were conducted in November of 2023 on days coinciding with fish habiatat assessments in Weigert Creek.  Measurements were taken at points upstream and dowstream of Highway 43 at both Weigert Creek and Brule Creek.  
```

Comment:
We should incorperate the writing that I did vs throw it away.  It has information as important (or more) than anything else

Reply:
oh maybe it is there.